### ISSUE TITLE
Warning text needs more contrast

### ISSUE THREAD

**Issue by angelamnr** [thememachine/eleven#156]
# The issue is:
Warning text don't have enough contrast with the message warning background color.
Message text color is yellowsnow (#f5ba41) and the background color is yellowsnow-light (#fcce54). [WCAG contrast ](http://webaim.org/resources/contrastchecker/) for this is 1.18:1 when WCAG AA compliance requires 4.5:1.

# Screenshots:
http://i.imgur.com/P4xZ7YO.png

# Where (url)
Any page with an admin warning message.
https://github.com/thememachine/eleven/blob/8.x-1.x/eleven/templates/component/messages/messages.css#L73-L76

Todo
- [ ] fix this
- [ ] fit that

**Comment by fuzzbomb** [comment]
The link text in warning messages doesn't have good contrast either.

yellowsnow on yellowsnow-light is 1.2:1
https://leaverou.github.io/contrast-ratio/#rgb%28245%2C%20186%2C%2065%29-on-rgb%28252%2C%20206%2C%2084%29

bluesky on yellowsnow-light is 1.8:1
https://leaverou.github.io/contrast-ratio/#rgb%2824%2C%20169%2C%20224%29-on-rgb%28252%2C%20206%2C%2084%29

**Comment by mortendk** [comment]
![image](https://user-images.githubusercontent.com/65756/29069265-819c4a16-7c3a-11e7-9a57-089aca056bb0.png)

**Comment by fuzzbomb** [comment]
The warning message text colours still don't meet WCAG AA.  The contrast of --pupkin on --yellowsnow-light is only 3.8:1

https://leaverou.github.io/contrast-ratio/#rgb%28195%2C%2068%2C%2033%29-on-rgb%28254%2C%20222%2C%20136%29
